### PR TITLE
Feature/restyle

### PR DESCRIPTION
**Pull request by nii236**
screen shot 2016-11-16 at 21 16 40
screen shot 2016-11-16 at 21 16 35
screen shot 2016-11-16 at 21 16 30
screen shot 2016-11-16 at 21 16 12
screen shot 2016-11-16 at 21 15 52
screen shot 2016-11-16 at 21 15 48

### CODE REVIEW

File: web/templates/user/form.html.haml [hunk 1]
```diff
@@ -15,4 +15,4 @@
     = password_input f, :password_confirmation, placeholder: "confirm password", class: "br2 ba b--silver pa3 tc w-100 mv1"
     .red.tc= error_tag f, :password_confirmation
   .form-group
-    = submit "Create account", class: "pa3 f5 br2 bg-white ba b--black center db w-third mt4"
+    = submit "Create account", class: "pa3 f5 br2 bg-blue ba b--black center db w-third mt4 light-gray b--transparent"
```
Comment (jgunnink) [inline web/templates/user/form.html.haml]: When editing a user we should contextually show edit. When creating it should show create.

File: web/templates/session/form.html.haml [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+.measure-narrow.center
+  - form_for @changeset, @action, fn f ->
+    - if f.errors != [] do
+      .red
+        %p Oops, something went wrong! Please check the errors below:
+        %ul
+          - for {attr, message} <- f.errors do
+            %li
+              = humanize(attr)
+              = message
```
Comment (jgunnink) [inline web/templates/session/form.html.haml:10]: 👍

File: web/templates/user/edit.html.haml [hunk 3]
```diff
@@ -1,5 +1,5 @@
-.measure-wide.center
+.measure.center
   %h2.tc Edit user
   = render "form.html", changeset: @changeset, action: user_path(@conn, :update, @user)
```
Comment (jgunnink) [inline web/templates/user/edit.html.haml]: Should this be a partial? The filename isn't prefixed with an underscore `_`. Also is the .html part on this line necessary?

**Review by jgunnink — APPROVED** [review]
Looks good. Just fix up that form saying "Create" when editing a user.

File: web/templates/user/edit.html.haml (hunk 3 above)
Comment (nii236) [reply web/templates/user/edit.html.haml]: I've never seen underscores in partials in phoenix land. Best to stick with the idioms

Comment (nii236) [reply web/templates/user/edit.html.haml]: `.html` is needed

**Pull request merged**